File: openapi-generator/dart-jaguar/lib/model/fleet_vehicle_location.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';

part 'fleet_vehicle_location.jser.dart';

class FleetVehicleLocation {
   /* The latitude of the location in degrees. */
  @Alias('latitude')
  final double latitude;
   /* The best effort (street,city,state) for the latitude and longitude. */
  @Alias('location')
  final String location;
   /* The longitude of the location in degrees. */
  @Alias('longitude')
  final double longitude;
   /* The speed calculated from GPS that the asset was traveling at in miles per hour. */
  @Alias('speedMilesPerHour')
  final double speedMilesPerHour;
   /* Time in Unix milliseconds since epoch when the asset was at the location. */
  @Alias('timeMs')
  final int timeMs;
  

  FleetVehicleLocation(
      

{
     this.latitude = null,  
     this.location = null,  
     this.longitude = null,  
     this.speedMilesPerHour = null,  
     this.timeMs = null 
    
    }
  );

  @override
  String toString() {
    return 'FleetVehicleLocation[latitude=$latitude, location=$location, longitude=$longitude, speedMilesPerHour=$speedMilesPerHour, timeMs=$timeMs, ]';
  }
}

@GenSerializer(nullableFields: true)
class FleetVehicleLocationSerializer extends Serializer<FleetVehicleLocation> with _$FleetVehicleLocationSerializer {

}

File: openapi-generator/scala-gatling/src/gatling/scala/org/openapitools/client/model/FleetVehicleLocation.scala

package org.openapitools.client.model


case class FleetVehicleLocation (
    /* The latitude of the location in degrees. */
    _latitude: Option[Double],
    /* The best effort (street,city,state) for the latitude and longitude. */
    _location: Option[String],
    /* The longitude of the location in degrees. */
    _longitude: Option[Double],
    /* The speed calculated from GPS that the asset was traveling at in miles per hour. */
    _speedMilesPerHour: Option[Double],
    /* Time in Unix milliseconds since epoch when the asset was at the location. */
    _timeMs: Option[Long]
)
object FleetVehicleLocation {
    def toStringBody(var_latitude: Object, var_location: Object, var_longitude: Object, var_speedMilesPerHour: Object, var_timeMs: Object) =
        s"""
        | {
        | "latitude":$var_latitude,"location":$var_location,"longitude":$var_longitude,"speedMilesPerHour":$var_speedMilesPerHour,"timeMs":$var_timeMs
        | }
        """.stripMargin
}
